hs_core/tests/api/rest/test_create_resource.py

```diff
@@ -36,3 +37,275 @@ def test_post_resource_get_sysmeta(self):
+        statement = 'This resource is shared under the Creative Commons Attribution CC BY.'
+        url = 'http://creativecommons.org/licenses/by/4.0/'
+        metadata.append({'rights': {'statement': statement, 'url': url}})
+        metadata.append({'language': {'code': 'fre'}})
+
+        # contributor
+        con_name = 'Mike Sundar'
+        con_org = "USU"
+        con_email = 'mike.sundar@usu.edu'
+        con_address = "11 River Drive, Logan UT-84321, USA"
+        con_phone = '435-567-0989'
+        con_homepage = 'http://usu.edu/homepage/001'
+        metadata.append({'contributor': {'name': con_name,
+                                         'organization': con_org, 'email': con_email,
+                                         'address': con_address, 'phone': con_phone,
+                                         'homepage': con_homepage}})
+
+        # creator
+        cr_name = 'John Smith'
+        cr_org = "USU"
+        cr_email = 'jsmith@gmail.com'
+        cr_address = "101 Clarson Ave, Provo UT-84321, USA"
+        cr_phone = '801-567=9090'
+        cr_homepage = 'http://byu.edu/homepage/002'
+        metadata.append({'creator': {'name': cr_name, 'organization': cr_org,
+                                     'email': cr_email, 'address': cr_address,
+                                     'phone': cr_phone, 'homepage': cr_homepage}})
+
+        # relation
+        metadata.append({'relation': {'type': 'isPartOf',
+                                      'value': 'http://hydroshare.org/resource/001'}})
+        # source
+        metadata.append({'source': {'derived_from': 'http://hydroshare.org/resource/0001'}})
+
+        # identifier
+        metadata.append({'identifier': {'name': 'someIdentifier', 'url': 'http://some.org/001'}})
+
+        # fundingagency
+        agency_name = 'NSF'
+        award_title = "Cyber Infrastructure"
+        award_number = "NSF-101-20-6789"
+        agency_url = "http://www.nsf.gov"
+        metadata.append({'fundingagency': {'agency_name': agency_name, 'award_title': award_title,
+                                           'award_number': award_number, 'agency_url': agency_url}})
+        params = {'resource_type': rtype,
+                  'title': title,
+                  'metadata': json.dumps(metadata),
+                  'file': ('cea.tif',
+                           open('hs_core/tests/data/cea.tif'),
+                           'image/tiff')}
+        rest_url = '/hsapi/resource/'
+        response = self.client.post(rest_url, params)
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+        content = json.loads(response.content)
+        res_id = content['resource_id']
+        resource = get_resource_by_shortkey(res_id)
+        self.assertEqual(resource.metadata.coverages.all().count(), 1)
+        self.assertEqual(resource.metadata.coverages.filter(type='period').count(), 1)
```

Comment:
@pkdash Perhaps the test needs to further verify period temporary coverage values are also correct here?